subsys/caf/modules/sensor_sampler.c

```diff
@@ -302,9 +357,47 @@ static int sensor_trigger_init(const struct sensor_config *sc, struct sensor_dat
 		return -ENOMEM;
 	}
 
+	reset_sensor_trigger_counter(sc, sd);
+
+	LOG_INF("Trigger configured (threshold: %d, time: %d ms)",
+		(int)(1000 * sc->trigger->activation.thresh),
+		sc->trigger->activation.timeout_ms);
 	return 0;
 }
 
+static void configure_max_power_state(void)
```

Comment:
This is hard to understand. Could you provide more extensive comments in the code?